proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java

```diff
@@ -254,7 +261,21 @@ public void handle(Handshake handshake) throws Exception
         this.handshake = handshake;
         ch.setVersion( handshake.getProtocolVersion() );
 
-        // Starting with FML 1.8, a "\0FML\0" token is appended to the handshake. This interferes 
+        if ( handshake.getRequestedProtocol() == 2 && BungeeCord.getInstance().getConnectionThrottle().throttle( ( ( InetSocketAddress ) ch.getHandle().remoteAddress() ).getAddress() ) )
```

Comment:
As magic as in the switch below. Should I add another comment here that the two means?